wordpress-mobile/WordPress-iOS
# Stock photos. Step II. Start scaffolding an implementation based on WPMediaPicker

Second step in the implementation of #8954. If we agree to continue exploring this path, this could be a starting point.

In this PR:
- Add classes necessary to present an empty WPMediaPicker: a data source, a media item, and an empty results view. All those classes are just placeholders, that do not provide any meaningful implementation yet.
- Add `StockPhotosPicker` which is responsible for presenting the WPMediaPicker. 

To test:
1. Start creating a new post
2. Tap the + button in the lower Format Bar.
3. Tap the ellipsis in the Format Bar
4. There should be an alert with three options (Free photo Library, Files, Cancel)
5. Selecting Free Photos Library should dismiss the alert and present this:
![simulator screen shot - iphone 8 - 2018-03-28 at 17 26 58](https://user-images.githubusercontent.com/2722505/38020617-4bc7fef6-32ad-11e8-83ee-2ec03b949c8c.png)

That is  an instance of `WPNavigationMediaPickerViewController`. In this PR we have provided just enough to be able to present it, without being able to render any content yet (other than the no results message)
